resources/lang/fi/store.php

```diff
@@ -105,7 +105,7 @@
     'supporter_tag' => [
         'gift' => 'lahjoita pelaajalle',
         'require_login' => [
-            '_' => 'Ollaksesi Tukija sinun täytyy olla :link!',
+            '_' => '',
```

Comment:
Some of the translations got removed like this one, but I can't seem to find any history of the change

Reply:
Yeah this one was available to translate on crowdin for no reason

Reply:
original translation has been changed so looks like crowdin assumes it's a different thing.

Reply:
Oh right i think it was after every `supporter tag` was changed into `osu!supporter tag`